docs/preview/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Home"
+layout: default
+permalink: /
```

Comment:
Let's remove all permalinks & redirects

Reply:
This is the preview, so, will be placed to the root index after the release. I think it's ok here. Removed the links and redirects in the older versions though.